locale/zh-tw/security.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: security.hbs
+title: 安全
+---
+
+# 安全
+
+## 回報 Node.js 的臭蟲
+
+Node.js 非常嚴肅地看待安全性臭蟲，所有的臭蟲都應回報至 [HackerOne](https://hackerone.com/nodejs)，其後將由專責安全問題的部門處理。
+
+我們將在 24 小時內確認你的回報，並於 48 小時內回覆你後續處置的相關細節。
+
+向你回覆初步的進展後，安全團隊將會持續讓你了解修補狀況及全面公告的進展，他們可能會向你詢問進一步的細節或尋求你對此問題的建議。我們至少每五天就會向你更新進展，通常每隔約 24-48 小時就會更新一次。
+
+### Node.js 臭蟲獎賞計劃
+
+Node.js 官方為了安全研究員們及負責的公開揭露而展開了一個臭蟲獎賞計劃。我們使用 HackerOne 管理計畫，詳情請見 [https://hackerone.com/nodejs](https://hackerone.com/nodejs)。
+
+## 回報三方模組中的臭蟲
```

Comment:
This is wrong, I suggest you to check it again, I can provide an "IMO correct one" if you have different opinion with it.

Reply:
That will be great. I would like to hear you opinion before I make any change :)

Reply:
"Third" party, not "Three" party

Reply:
Oh, I thought you were reviewing "Node.js Bug Bounty Program" 🤦‍♂️
Fixed now.